docs/generated/devkit/ProjectConfiguration.md

```diff
@@ -100,6 +100,7 @@ Project specific configuration for `nx release`
 
 | Name       | Type                                                                                                                                                                                                                                                                                                                                                                                     |
 | :--------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `docker?`  | `true` \| `NxReleaseDockerConfiguration`                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
I don't know why this only failed `nx documentation` during the migration and not eariler.

Reply:
weird. approved it for now at least